reframe/frontend/executors/policies.py

```diff
@@ -98,8 +96,8 @@ def runcase(self, case):
         check, partition, environ = case
 
         self.printer.status(
-            'RUN', '%s on %s using %s' %
-            (check.name, partition.fullname, environ.name)
+            'RUN',
+            f'{check.name} on {partition.fullname} using {environ.name}'
```

Comment:
Actually, using `check.info()` should make it forward compatible; so better use that.

Reply:
Just a small detail. `check.info()` has the full information (partition and environment after the setup. And I think it's not nice to print the message after the setup, in case of failure. Is there a way to get this from self._case at this point? https://github.com/eth-cscs/reframe/blob/master/reframe/core/pipeline.py#L1188